docs/application/native/guides/connectivity/inm.md

```diff
@@ -0,0 +1,171 @@
+# INM
+
+
+INM means Intelligent Network Monitoring. It can be used to get information about the network provided by well-known linux functions. Some features provided by this API are: detecting IP collision, dumping network status, dumping TCP, monitoring ethernet, monitoring Wi-Fi module state and getting network statistics.
+
+This feature is supported in mobile, tv and wearable profile.
+
+> **Note**  
+> You can test the INM functionality on a target device only. The [emulator](../../../tizen-studio/common-tools/emulator.md) does not support this feature.
+
+## Prerequisites
+
+To enable your application to use the INM API:
+
+1. To use the INM API, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+   ```
+   <privileges>
+      <privilege>http://tizen.org/privilege/network.get</privilege>
+      <privilege>http://tizen.org/privilege/network.set</privilege>
+   </privileges>
+   ```
+
+2. To use the functions and data types of the INM API, include the <inm.h> header file in your application:
+
+   ```
+   #include <inm.h>
+   ```
+
+3. Call the `inm_initialize()` function to initialize library:
+
+    ```
+    inm_initialize();
+    ```
+
+4. Call the `stc_deinitialize()` function to deinitialize library before quiting:
```

Comment:
Kindly remove “the” and “function” word while mentioning a function. Also a suggestion to avoid using the word “quitting” to align with other Tizen guides, can we rephrase as:

When no longer needed, call `stc_deinitialize()` to deinitialize the library:

Please verify and share your thoughts on the same before updating.

Reply:
Yes, this does seem better. Updated.